tests/acceptance/upcoming-project-cards-renders-test.js

```diff
@@ -33,13 +33,15 @@ module('Acceptance | upcoming project cards renders', function(hooks) {
         this.server.create('assignment', {
           id: 1,
           tab: 'upcoming',
-          dcpPublicstatus: 'Filed',
```

Comment:
i like having this here though because it makes it explicit to future test readers that this is testing for behavior that's downstream of the value of `dcpPublicstatus`, no?

Reply:
oh wait it's not a real field is it? I don't even think the API returns a dcpPublicstatus... just a dcpPublicstatusSimp

Reply:
@allthesignals API does return `dcpPublicstatus`, but on staging it was returning `null` or `""`, so Hannah and I decided to revert it back to `dcpPublicstatusSimp`